lib/webpack.js.js

```diff
@@ -76,9 +76,7 @@ module.exports = (config, options) => {
     if (options.minify) {
       const Terser = require('terser-webpack-plugin')
```

Comment:
Is this necessary? https://webpack.js.org/plugins/terser-webpack-plugin/#getting-started suggests that v5 has terser built in

Reply:
Looks like it
![image](https://user-images.githubusercontent.com/324440/152616695-ba2c1202-35ec-4812-8cfa-21e290257721.png)